org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/servlet/AbstractServletTest.java

```diff
@@ -173,15 +173,16 @@ public void disconnect()  throws Exception {
 			servlet.disconnect();
 		}
 	}
-	
+
+	@SuppressWarnings("sonar:S1192")
 	protected ScanBean createStepScan() throws IOException {
 		// We write some pojos together to define the scan
 		final ScanBean bean = new ScanBean();
 		bean.setName("Hello Scanning World");

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Hello Scanning World" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)